lib/composites/modals/style.less

```diff
@@ -5,11 +5,11 @@
   width: 100%;
   .box-sizing(border-box);
 
-  &@{prefix}modal-show {
```

Comment:
if this is in effort to share styles between the modal and alert, lets use "dialog" as the convention here (they're both dialogs)...`@{prefix}dialog-show`, `@{prefix}dialog-inner`